server/src/main/java/org/apache/druid/server/log/FileRequestLoggerProvider.java

```diff
@@ -58,8 +58,12 @@
   @Override
   public RequestLogger get()
   {
-    FileRequestLogger logger = new FileRequestLogger(jsonMapper, factory.create(1, "RequestLogger-%s"), dir,
-        filePattern);
+    FileRequestLogger logger = new FileRequestLogger(
+            jsonMapper,
```

Comment:
In Druid, the wrapping indentation is 4, not 8 spaces. Please also fix other places in this PR.